quickwit/quickwit-cli/src/index.rs

```diff
@@ -904,6 +908,11 @@ pub async fn ingest_docs_cli(args: IngestDocsArgs) -> anyhow::Result<()> {
         let index_service = IndexService::from_config(config.clone()).await?;
         index_service.clear_index(&args.index_id).await?;
     }
+    // The indexing service needs to update its cluster chitchat state so that the control plane is
+    // aware of the running tasks. We thus create a fake cluster to instantiate the indexing service
+    // and avoid impacting potential control plane running on the cluster.
+    let transport = ChannelTransport::default();
+    let fake_cluster = create_cluster_for_test(Vec::new(), &["indexer"], &transport, false).await?;
```

Comment:
Is this the only reason for chitchat dependency? 
Could we either re-export the transport symbol in quickwit-cluster or add a method to create a test cluster using a `channel::Transport::default()`